GitWorkflow/GitWorkflow/main.cpp

```diff
@@ -20,7 +25,10 @@ int main()
 
 		window.clear();
 		player.update();
+		if (player.getGlobalBounds().intersects(endPoint.getGlobalBounds())) { window.close(); }
```

Comment:
Separate update code from rendering code.
Update and intersection, then clear and draw.